book/src/ui-configuration.md

```diff
@@ -9,23 +9,20 @@ following configuration screen.
 
 ## Connecting to the Clients
 
-This allows you to enter the address and ports of the associated Lighthouse
+Both the Beacon node and the Validator client need to have their HTTP APIs enabled. These ports should be accessible from the computer running Siren. This allows you to enter the address and ports of the associated Lighthouse
 Beacon node and Lighthouse Validator client.
 
-> The Beacon Node must be run with the `--gui` flag set. 
+To enable the HTTP API for the beacon node, utilize the `--gui` CLI flag. This action ensures that the HTTP API can be accessed by other software on the same machine.
 
-If you run Siren in the browser (by entering `localhost` in the browser), you will need to allow CORS in the HTTP API. This can be done by adding the flag `--http-allow-origin "*"` for both beacon node and validator client. If you would like to access Siren beyond the local computer, we recommend using an SSH tunnel. This requires a tunnel for 3 ports: `80` (assuming the port is unchanged as per the [installation guide](./ui-installation.md#docker-recommended), `5052` (for beacon node) and `5062` (for validator client). You can use the command below to perform SSH tunneling:
-```bash
-ssh -N -L 80:127.0.0.1:80 -L 5052:127.0.0.1:5052 -L 5062:127.0.0.1:5062 username@local_ip
-```  
+> The Beacon Node must be run with the `--gui` flag set.
 
-where `username` is the username of the server and `local_ip` is the local IP address of the server. Note that with the `-N` option in an SSH session, you will not be able to execute commands in the CLI to avoid confusion with ordinary shell sessions. The connection will appear to be "hung" upon a successful connection, but that is normal. Once you have successfully connected to the server via SSH tunneling, you should be able to access Siren by entering `localhost` in a web browser. 
+If you require accessibility from another machine within the network, configure the `--http-address` as `0.0.0.0`.
 
-You can also access Siren using the app downloaded in the [Siren release page](https://github.com/sigp/siren/releases). To access Siren beyond the local computer, you can use SSH tunneling for ports `5052` and `5062` using the command:
+> To access from another machine on the same network set the Beacon Node and Validator Client `--http-address` as `0.0.0.0`.
```

Comment:
I think this conflicts with the FAQ section, although both approach should work. It's probably best to recommend listening to the LAN ip address than 0.0.0.0 (all interfaces).

> ## 4. How do I connect Siren to Lighthouse from a different computer on the same network?
> The most effective approach to enable access for a local network computer to Lighthouse's HTTP API ports is by configuring the `--http-address` to match the local LAN IP of the system running the beacon node and validator client. For instance, if the said node operates at `192.168.0.200`, this IP can be specified using the `--http-address` parameter as `--http-address 192.168.0.200`.
Subsequently, by designating the host as `192.168.0.200`, you can seamlessly connect Siren to this specific beacon node and validator client pair from any computer situated within the same network.

Reply:
Agree with @jimmygchen that it is better to just listen to the local IP address `--http-adress local_IP` instead of `--http-address 0.0.0.0`

Reply:
I agree too. Ultimately this is going to come down to the people that are supporting our users. Not sure how many networks our users get into, but I often have multiple lan ip addresses (could just be me, also ipv6 addresses). Some users may also not know how to find their local LAN ip. 

The `0.0.0.0` approach is the simplest to get it working for users although not the best. If the people helping out on discord are happy to help potential confused users, go for it. It might be worth mentioning there is the option of using `0.0.0.0` if you don't know what a local LAN IP is.

Reply:
alright, in the highlight i can put the local lan ip and in the text mention that you can default to 0.0.0.0 but its not recommended